diesel/src/expression/expression_methods/escape_expression_methods.rs

```diff
@@ -23,7 +23,7 @@ use types::VarChar;
 /// #     use diesel::insert;
 /// #     let connection = establish_connection();
 /// #     insert(&NewUser { name: "Ha%%0r".into() }).into(users)
-/// #         .execute(&connection).unwrap();
+/// #         .try!(execute(&connection));
```

Comment:
😢 Well that's a bummer. 

Thanks for the pointer! Pushing up the change in a few.